packages/react-conformance/tsconfig.json

```diff
@@ -7,15 +7,17 @@
     "jsx": "react",
     "declaration": true,
     "sourceMap": true,
+    "noImplicitAny": true,
```

Comment:
You can get rid of the explicit settings for `noImplicitAny` and `noImplicitThis` because both of those are implied by `strict: true`.  (see the entry for `--strict` [here](https://www.typescriptlang.org/docs/handbook/compiler-options.html))